core/shark_turbine/aot/exporter.py

```diff
@@ -178,6 +179,8 @@ def export(
       dynamic_shapes: Dynamic shape specs to pass to torch.export.
       external_params: Whether to declare parameters as external vs inlining
         contents.
+      global_params: Whether to promote module parameters to globals eagerly
```

Comment:
Is there perhaps an opt out marker too? (E g., this is either none + opt in or all, wondering if all + opt out option)

Reply:
Yeah, this entry point is getting out of control. I'm going to refactor it before landing into a few overloads and better parameterized some of it. There should be fewer options needed if taking the approach of just annotating what you want on prior to export and then having export always so the same thing.

Reply:
(i just added this to text the path with no magic, which is what I want to be the default)

Reply:
Cleaned all of this up.